app/parse.py

```diff
@@ -14,9 +36,98 @@ class Product:
     rating: int
     num_of_reviews: int
 
+    @classmethod
+    def product_fields(cls) -> list[str]:
+        return [field.name for field in fields(cls)]
+
+
+class Scraper:
+    def __init__(self) -> None:
+        self.driver = ChromeDriver()
+
+    @staticmethod
+    def parse_single_product(product_soup: BeautifulSoup) -> Product:
+        return Product(
+            title=product_soup.select_one(".title")["title"],
+            description=product_soup.select_one(
+                ".description"
+            ).text.replace("\xa0", " "),
+            price=float(
+                product_soup.select_one(".price").text.replace("$", "")
+            ),
+            rating=int(len(product_soup.find(
+                "div", class_="ratings"
+            ).find_all("span", class_="ws-icon"))),
+            num_of_reviews=int(
+                product_soup.select_one(".review-count").text.split()[0]
```

Comment:
if it's a class, then it's better to create a separate method that has implementation of scraping specific field